src/api/prevalence/content-types/prevalence/schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "kind": "collectionType",
+  "collectionName": "prevalences",
+  "info": {
+    "singularName": "prevalence",
+    "pluralName": "prevalences",
+    "displayName": "Prevalence"
+  },
+  "options": {
+    "draftAndPublish": false
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "samplingYear": {
+      "type": "integer"
+    },
+    "zomoProgram": {
+      "type": "string",
+      "private": true
+    },
+    "furtherDetails": {
+      "type": "text"
+    },
+    "numberOfSamples": {
+      "type": "string"
+    },
+    "numberOfPositive": {
```

Comment:
Should be an integer

Reply:
I saw this weird value (4,3–49,0) in Positive_Proben_N columns. thus I thought it is better to keep everything string.  I thought I'll check with Carolina if that is valid value or a mistake